src/types.rs

```diff
@@ -240,6 +240,65 @@ impl BlockTime {
     }
 }
 
+/// Balance differentiated in various categories
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone, Default)]
+pub struct Balance {
+    /// All coinbase outputs not yet matured
+    pub immature: u64,
+    /// Unconfirmed UTXOs generated by a wallet tx
+    pub trusted_pending: u64,
+    /// Unconfirmed UTXOs received from an external wallet
+    pub untrusted_pending: u64,
+    /// Confirmed and immediately spendable balance
+    pub available: u64,
+}
+
+impl Balance {
+    /// Get sum of trusted_pending and available coins
+    pub fn get_trusted(&self) -> u64 {
```

Comment:
If we rename `available -> confirmed`, we can call this method `get_spendable`.

Would this be clearer? May need to ask everyone else's opinion.

Reply:
ACK for get_spendable :)